apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/IterableContainsSpecBase.kt

```diff
@@ -37,6 +37,7 @@ abstract class IterableContainsSpecBase {
     protected val only = IterableContains.Builder<*, *, InAnyOrderSearchBehaviour>::only.name
     protected val inOrderOnlyValues = IterableContains.Builder<Int, Iterable<Int>, InOrderOnlySearchBehaviour>::values.name
     protected val inOrderOnlyEntries = IterableContains.Builder<Int, Iterable<Int>, InOrderOnlySearchBehaviour>::entries.name
+    protected val elementsOf = IterableContains.Builder<Int, Iterable<Int>, InOrderOnlySearchBehaviour>::elementsOf.name
```

Comment:
maybe you renamed this due to my last comment. I prefer if this is named inOrderElementsOf as there will be an inAnyOrderElementsOf as well